templates/components/products/carousel.html

```diff
@@ -40,5 +40,8 @@
     {{/each}}
     {{#if products.length '>' 1}}
     <button aria-label="{{lang 'carousel.arrowAriaLabel'}} {{products.length}}" class="{{#if customArrowClass}}{{customArrowClass}} {{/if}}js-product-next-arrow slick-next slick-arrow"></button>
+    <span class="carousel-content-change-message aria-description--hidden" role="status">
```

Comment:
as this class used for logic but not for styles, can we add "js-" prefix to it as it is done for example for play/pause button?

Reply:
np